package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
```

Comment:
the `main` property is a hook into node modules system, must be a fully qualified path. It's not a structured run component that's paired with the value passed to `electron` via the `start` script.

the `main` is the file that would be required in if your module was used programmatically.